src/utils/utils.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+export const getIdx = (arr, match) => {
```

Comment:
There's a javascript function that implements this kind of behavior: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex

Reply:
Thanks a lot Erick, would check into that! Thanks!